enclave-tls/src/enclave_quotes/sgx-ecdsa/verify_evidence.c

```diff
@@ -42,6 +42,13 @@ enclave_quote_err_t sgx_ecdsa_verify_evidence(enclave_quote_ctx_t *ctx,
 	ETLS_DEBUG("quote size is %d, quote signature_data_len is %d\n",
 		   quote_size, pquote->signature_data_len);
 
+	// First verify the hash value
+	int result = memcmp(hash, pquote->report_body.report_data.d, hash_len);
+	if (result != 0) {
```

Comment:
result is not used later so it is not necessary to define it. Just simply being `if (memcmp(...))` is enough.